src/pages/public_registration/PublicPublicationContext.tsx

```diff
@@ -520,7 +520,7 @@ const PublicAudioVisualPublicationDialogContent = ({
 
 const PublicConcertDialogContent = ({ concert }: { concert: Concert }) => {
   const { t } = useTranslation();
-  const { type, place, time, extent, concertProgramme, partOfSeries } = concert;
+  const { type, place, time, extent, concertProgramme, concertSeries: partOfSeries } = concert;
```

Comment:
Bare fjern all bruk av det gamle navnet for å unngå forvirring
```suggestion
  const { type, place, time, extent, concertProgramme, concertSeries } = concert;
```